src/Sulu/Component/Webspace/Manager/WebspaceCollectionBuilder.php

```diff
@@ -146,7 +139,7 @@ function ($a, $b) {
             );
         }
 
-        $collection->setWebspaces($this->webspaces);
+        $collection->setWebspaces($webspaces);
         $collection->setPortals($this->portals);
         $collection->setPortalInformations($this->portalInformations);
```

Comment:
I wonder if $portalInformations and $portals can also be removed from the class scope.